moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchInput.js

```diff
@@ -152,10 +154,16 @@ const NavigationSearchInput = ({ input404Class }) => {
   const [showSlideIn, toggleSlideIn] = useState(false);
 
   const [loadSearchResults, { called, loading, data }] = useLazyQuery(
-    projectSearchQuery.gql,
-    projectSearchQuery.config.options.useQuery
+    projectSearchQuery.current.gql,
+    projectSearchQuery.current.config.options.useQuery
   );
 
+  /* Clear input and reset the gqlabstract 'or' config */
+  const clearSearchInput = () => {
```

Comment:
I combined these two actions since they happen a lot. We could also change up what happens between uses of the search here too.

Reply:
thanks for this! i agree that closing the search input should clear the results. this is an improvement over the production behavior which confusingly persists the previous search resuts! 


out of scope, but i think the ideal behavior is that clicking on a search result hides the search results menu, but does not hide or clear the search input. that way you can click back into the search input to navigate to another item in your result. i'll open a separate issue for that :)

Reply:
awesome - that sounds great 👍